### DISCUSSION TITLE
Highlight pages that need updating

### DISCUSSION THREAD

**Discussion by Rumyra** [mdn/mdn-community#547] [Content]
This came up in in the editorial meeting:

It would be good to have a flag on pages to indicate if the information is out of date, this can be because

* The spec has changed since it was last edited
* The page hasn't been updated for a specified period of time and so probably could do with checking

This information would be used as a highlight when working locally on mdn/content (much like the current flaws info) *not* on the live site. It could also be used to parse a number of documents to check if any of them need updating.

This discussion is to consider different ways in which we can implement. Is it simply we have a `last-edited` item of frontmatter, or something more involved (like checking changes in w3c specs against current docs)

**Comment by teoli2003** [comment]
For the spec has changed since it was last edited, we should compare with changes to the part of the spec linked in front-runner.

Because some large specs have changes every single day (almost).

The question here is also: how to flag (locally and remotely) that spec changes up-to-now have been checked and the page doesn't need new update. A lot of spec changes doesn't lead to a doc change.

**Comment by hamishwillee** [comment]
A "TODO" macro is one approach used for this kind of thing (e.g. Sphinx SSG). These display as warning blocks in local/dev builds only. The infrastructure allows listing of all pages with blocks that have the macro.

Perhaps we might use part of this idea? Specifically, we could use frontmatter rather than a macro to generate the big fat local warning and lists of affected articles. 

This resonates: "how to flag (locally and remotely) that spec changes up-to-now have been checked and the page doesn't need new update.". Perhaps a field with "last spec checked" that could be compared to the BCD to dynamically generate the "something changed that needed an update".
